### ISSUE TITLE
Intel 9560

### ISSUE THREAD

**Issue by matschiner** [rpeshkov/IntelWifi#17]
Hi,
is there anyone who got a Intel 9560 to work?
Greetings
Max

**Comment by radeno** [comment]
I think this should
https://github.com/Rashed97/IntelWifi

**Comment by Weddin** [comment]
Nope. Rashed97 not resolved the initialization. Only create classes for future.

**Comment by DarkFighterLuke** [comment]
I have this card and I am interested in.
Any solution at the moment?
